receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go

```diff
@@ -34,18 +35,26 @@ const metricsLen = 3
 type scraper struct {
 	logger *zap.Logger
 	config *Config
+	mb     *metadata.MetricsBuilder
 
 	// for mocking
-	load func() (*load.AvgStat, error)
+	bootTime func() (uint64, error)
```

Comment:
Nit: looks like `bootTime` actually not being used so I would recommend removing it until it's needed

Reply:
fixed. Added a unit test.